src/main/java/SpaceRemoval.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class SpaceRemoval {
```

Comment:
Merge this class with another, which is doing similar things (reducing form redundant symbols/lines)